beacon_node/network/src/beacon_processor/mod.rs

```diff
@@ -557,6 +559,22 @@ impl<T: BeaconChainTypes> WorkEvent<T> {
         }
     }
 
+    /// Create a new work event to process `LightClientBootstrap`s from the RPC network.
+    pub fn lightclient_bootstrap_request(
+        peer_id: PeerId,
+        request_id: PeerRequestId,
+        request: LightClientBootstrapRequest,
+    ) -> Self {
+        Self {
+            drop_during_sync: false,
```

Comment:
I think we can be aggressive about dropping these during sync, seeing as they require a non-trivial amount of work

```suggestion
            drop_during_sync: true,
```